recipes/scmdata/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: netcdf-scm
```

Comment:
There is already a feedstock named `netcdf-scm`, https://github.com/conda-forge/netcdf-scm-feedstock, thus the linting message.

Reply:
The package name should be `scmdata`.

Reply:
I am so stupid... thanks!